exonum/src/blockchain/service.rs

```diff
@@ -340,6 +341,18 @@ impl SharedNodeState {
         }
     }
 
+    /// Is the node enabled?
+    pub fn is_enabled(&self) -> bool {
+        let state = self.state.read().expect("Expected read lock.");
+        state.is_enabled
+    }
+
+    /// Informs internal state about node's halting.
+    pub fn update_is_enabled(&self, is_enabled: bool) {
+        let mut state = self.state.write().expect("Expected read lock.");
+        state.is_enabled = is_enabled;
+    }
```

Comment:
Similar here, `enabled/set_enabled`.

I am not sure if the styleguide required to have the `is_` prefix... At least in Cargo, we don't use `is_`: https://github.com/rust-lang/cargo/blob/df11070cb574a64b167d36b012df50cf62e061b9/src/cargo/util/config.rs#L502-L516. However, we don't use `rustfmt` either 😅

Reply:
I like `is_` prefix. 😄 Plus we have things like `is_empty`, `is_char_boundary`, etc. in the standard library.

Reply:
I'll create another issue+PR for this.